R/annotation.R

```diff
@@ -80,7 +81,36 @@ annotate_ssm_blacklist = function(mutations_df,
   mutations_df = left_join(mutations_df,combined_blacklist,by = c("Chromosome", "Start_Position")) %>%
     mutate(blacklist_count = replace_na(blacklist_count, 0))
 
+  if (use_curated_blacklist){
+    repo_base = config::get("repo_base")
+    full_path = paste0(repo_base, config::get("etc")$curated_blacklist)
+    additional_blacklist = glue(full_path) %>% read_tsv()
+    additional_blacklist = additional_blacklist %>%
+      separate(chrpos, into = c("Chromosome", "Start_Position"), sep = ":")
+
+    additional_blacklist = mutate(additional_blacklist, Start_Position = as.integer(Start_Position)) %>%
+      group_by(Start_Position, Chromosome) %>%
+      summarize(blacklist_schmitz = sum(blacklist_schmitz)) %>%
+      ungroup()
+
+    if(str_detect(mutations_df$Chromosome, "chr")[1]){
```

Comment:
unnecessary if your black lists have the right prefixing for the projection. I'd leave this out and test that it works